README.md

```diff
@@ -535,6 +535,45 @@ curl -d '{ "jsonrpc": "2.0", "id": 5, "method": "getcontractstate", "params": ["
 
 List of supported by neo-go node RPC commands you can find [here](https://github.com/nspcc-dev/neo-go/blob/master/docs/rpc.md#supported-methods).
 
+#### CLI Utilities
+
+neo-go CLI provides `query tx` utility to check the transaction state. It uses
```

Comment:
@roman-khimov, we can't replace `getapplicationlog` usages by `query tx` command because we need stack and/or notifications for each example. Should we add `query tx` command invocation before each `getapplicationlog` invocation?

Reply:
That'd be somewhat duplicating, so we can leave it as it for now.